perf/load/auto-mtls/istio.sh

```diff
@@ -14,10 +14,17 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-pushd ../../istio-install || exit
-export ISTIO_RELEASE=${ISTIO_RELEASE:-"release-1.2-latest"}
-export DNS_DOMAIN=${DNS_DOMAIN:-"istio-automtls.local"}
-export VALUES="values-auto-mtls.yaml"
-./setup_istio.sh "${ISTIO_RELEASE}"
-popd || exit
+ENABLE_AUTO_MTLS=${ENABLE_AUTO_MTLS:-true}
```

Comment:
Instead of using a separate script, can we just reuse the existing installation script and add corresponding installation profile. Also we don't want to hardcode the release url.

Reply:
you mean `perf/istio-install/` ? Last time I tried and it's broken, the URL link schema for download Istio release seems has changed from 1.3 to 1.4.

Currently in HEAD, multiple shell exist, `setup_*operator`, setup_release.sh... All I need is being able to use `istioctl manifest --set global.mtls.auto=true`. is the `setup_operator.sh` the best to use?

Reply:
Done. Use the `setup_istio_operator.sh`. And made some changes correspondingly.